clang-tools-extra/clang-tidy/modernize/UseTrailingReturnTypeCheck.cpp

```diff
@@ -384,10 +385,15 @@ void UseTrailingReturnTypeCheck::keepSpecifiers(
 }
 
 void UseTrailingReturnTypeCheck::registerMatchers(MatchFinder *Finder) {
-  auto F = functionDecl(
-               unless(anyOf(hasTrailingReturn(), returns(voidType()),
-                            cxxConversionDecl(), cxxMethodDecl(isImplicit()))))
-               .bind("Func");
+  const auto hasNoWrittenReturnType =
```

Comment:
```suggestion
  const auto HasNoWrittenReturnType =
```
Please, follow LLVM naming convention, local variables starts with capital letters. `clang-tidy` can help you with that, run it on your changed files or enable it in `clangd`. The config is located at root of the project _llvm-project/.clang-tidy_

Reply:
Using clangd, but my user config overrode the clang-tidy checks from the project. Whoops.